docusaurus.config.js

```diff
@@ -140,7 +140,7 @@ module.exports = {
               className: "indent",
             },
             {
-              to: "sase/api/insights/2.0",
```

Comment:
Do you want to send them to 1.0 now?

Reply:
Yes. We're deprioritizing 2.0 for the time being.